### PR TITLE
Update to Jetty 11 and use of the jakarta.* packages

### PR DESCRIPTION
**Pull request by hlship**
This also removes support for tomcat; I put in a tag (`hls/prior-to-scrap-tomcat`) so that we can recover the deleted code, should we decide to resurrect it.

### CODE REVIEW

File: jetty/src/io/pedestal/http/jetty.clj [hunk 1]
```diff
@@ -177,23 +148,24 @@
...
         ssl (when (or ssl? ssl-port h2?)
               (ssl-conn-factory (assoc options :alpn alpn)))
         http-connector (when port
                          (doto (ServerConnector. server (into-array ConnectionFactory
+
                                                                     (remove nil? [http http2c])))
                            (.setReuseAddress reuse-addr?)
                            (.setPort port)
                            (.setHost host)))
+        ;; TODO: This is looking to be _very_ different in Jetty 11!
```
Comment (bonkydog) [inline jetty/src/io/pedestal/http/jetty.clj:168]: probably delete this now

**Review by bonkydog — APPROVED** [review]
Looks mostly great, but I left a few notes.

**Comment by brown131** [comment]
When will this be included in a new release?

**Comment by hlship** [comment]
Yes, it will likely be the focus of the 0.7 release, along with the new documentation.

**Pull request merged**